bot.py

```diff
@@ -160,6 +184,42 @@ async def watchlist_delete_media(interaction: nextcord.Interaction,
                                    media_name, watchlist_name):
     await delete_media(interaction, media_name, watchlist_name
                            )
+    
+# ---------------------------------------------------------------------------
+# Media Commands - Select a random media from watchlist
+# ---------------------------------------------------------------------------
+
+@bot.slash_command(guild_ids=[GUILD_ID],
+                   name="watchlist_choose",
+                   description="select a random item from a watchlist")
+async def watchlist_choose(interaction: nextcord.Interaction, watchlist_name):
+
+    watchlist_file = open(WATCHLISTFILENAME, 'r')
+    watchlist_data = json.load(watchlist_file)
+    watchlist_file.close()
+
+    watchlist_exists = False
+
+    for watchlist in watchlist_data["watchlists"]:
+        if watchlist["name"] == watchlist_name:
+            watchlist_exists = True
+            watchlist_length = len(watchlist["media"])
+            if watchlist_length == 0:
+                response = f"The **{watchlist_name}** watchlist is empty. \nYou can add items to it using '/watchlist_add <media_name> {watchlist_name}'"
+            else:
+                selected_media = random.choice(watchlist["media"])
+                
+                response = f"Selected media: **{selected_media}** \nTime to get out the popcorn!"
```

Comment:
Suggestion: Maybe change the response to "Lets watch {selected media} ......" so to sound more user friendly.